OpenRA.Game/Network/Session.cs

```diff
@@ -90,11 +90,16 @@ public string FirstEmptyBotSlot()
 			return Slots.FirstOrDefault(s => !s.Value.Closed && ClientInSlot(s.Key) == null && s.Value.AllowBots).Key;
 		}
 
-		public bool IsSinglePlayer
+		public bool IsSingleHuman
 		{
 			get { return Clients.Count(c => c.Bot == null) == 1; }
 		}
 
+		public IEnumerable<Session.Client> HumanPlayers
```

Comment:
Using "Human" here doesn't sit well with me.  There is no requirement for the other client to be human, just that it is another real game instance and not a bot ran on the same client.

How about `NonBotPlayers`?